jccli/jc_api_v1.py

```diff
@@ -46,11 +47,7 @@ def search_users(self, filter={}):
         :param filter: (dict) an object used to filter search results for various fields. E.g.: `{"firstname": "David"}`
```

Comment:
can you please mention that search can be done on multiple fields and it's an AND filter? An example would be nice as well.

Reply:
added